rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -61,11 +65,25 @@ def to_string(self, ts_converter: Callable[[Timestamp], str]) -> str:
 
         return desc
 
+    def _maybe_add_label_with_address(
+            self,
+            database: DBHandler,
+            matched_address: re.Match[str],
+    ) -> str:
+        """Adds a label of the address if it exists in a given string"""
+        chain_address = ChainAddress(
```

Comment:
THe docsstirng is not correct

```suggestion
        """Aux method to enrich addresses in the event notes using the addressbook""
        chain_address = ChainAddress(
```